### PR TITLE
Get rid off PolymerTemplate examples

### PR DESCRIPTION
**Pull request by szolo**
1) Remove PolymerTemplate examples.
2) Use `.ts` extension for LitElement.

### CODE REVIEW

File: documentation/polymer-templates/tutorial-template-and-binder.asciidoc [hunk 1]
```diff
@@ -10,13 +10,13 @@ layout: page
 
 In this section, we demonstrate how to create a fully-functional template-binder view.
 
-== Creating the Template Component
+It is recommended to use LitElement template language for your components, but PolymerTemplate (see <<tutorial-polymer-template-and-binder#,PolymerTemplate, Combining Templates and Binders>>) is also supported.
```
Comment (pleku) [inline documentation/polymer-templates/tutorial-template-and-binder.asciidoc:13]: I would not add this at all as the support is deprecated.
IMO it should go the other way around and polymer template docs should start with a disclaimer that it is recommended to use lit instead.

File: documentation/polymer-templates/tutorial-template-basic.asciidoc [hunk 2]
```diff
@@ -8,20 +8,18 @@ layout: page
 
 In this section, we demonstrate how to create a simple component using only the Template API.
 
-It is recommended to use LitElement template language for your components, but PolymerTemplate is also supported.
-
 Our example creates a simple view that allows the user to enter their name and click a button.
 
+It is recommended to use LitElement template language for your components, but PolymerTemplate (see <<tutorial-polymer-template-basic#,PolymerTemplate, Creating a Simple Component Using the Template API>>) is also supported.
```
Comment (pleku) [inline documentation/polymer-templates/tutorial-template-basic.asciidoc:13]: Same thing

File: documentation/polymer-templates/styling-polymer-templates.asciidoc [hunk 3]
```diff
@@ -8,11 +8,11 @@ layout: page
 
 Since client-side templates are Web Components, their content is inside shadow DOM. By design, shadow DOM defines a local style scope that is isolated from global styles. See <<../../themes/style-scopes#,Style Scopes>> for more information.
 
-== LitElement
-
 You can add component-specific scoped styles directly in the static `styles` template property.
 
-.`my-view.js`
+It is recommended to use LitElement template language for your components, but PolymerTemplate (see <<tutorial-polymer-template-basic#,PolymerTemplate, Styling Templates>>) is also supported.
```
Comment (pleku) [inline documentation/polymer-templates/styling-polymer-templates.asciidoc:13]: I would not add this as polymer template support is deprecated and to be removed

File: documentation/polymer-templates/tutorial-template-components-in-slot.asciidoc [hunk 4]
```diff
@@ -10,11 +10,11 @@ A client-side template cannot render child templates created using the `Componen
 
 To overcome this, you can add a `<slot>` element to mark the place where the light DOM elements should be rendered.
 
-== Using the <slot> Element
+It is recommended to use LitElement template language for your components, but PolymerTemplate (see <<tutorial-polymer-template-components-in-slot#,PolymerTemplate, Dynamically Adding Server-side Components to Templates>>) is also supported.
```
Comment (pleku) [inline documentation/polymer-templates/tutorial-template-components-in-slot.asciidoc:13]: Again, I would not add this

File: documentation/polymer-templates/tutorial-template-components-in-slot.asciidoc [hunk 5]
```diff
@@ -53,56 +53,11 @@ public class ComponentContainer extends LitTemplate {
...
-* Without the `<slot>` tag in the JavaScript Polymer template, the `label` and `button` would not be visible to the user, even though they can be located in the DOM.
-* You can add multiple components that will display in the slot when added to a template element with an open `<slot></slot>`.
-* You can remove any element from a `<slot>`. It will work as expected and no longer display in the main element.
-
 == Using Default Content in the <slot> Element
 
 The `<slot>` tag can contain default content that is displayed only if nothing else is set from the light DOM.
 
-*Example*: `<slot>` element with default content in a JavaScript client-side template.
+*Example*: `<slot>` element with default content in a TypeScript client-side template.
```
Comment (pleku) [inline documentation/polymer-templates/tutorial-template-components-in-slot.asciidoc:60]: Non-blocking: as both TS and JS are supported even though we demo only with TS, I think this is kind of unnecessary addition here and below

**Review by pleku — CHANGES_REQUESTED** [review]
Each polymer template documentation should start with the disclaimer that is in https://github.com/vaadin/flow-and-components-documentation/pull/1359 but these two PRs are in conclict. It makes sense to adopt that one on top of this one

**Comment by szolo** [comment]
@pleku I've addressed your comments and integrated PR from @denis-anisimov (https://github.com/vaadin/flow-and-components-documentation/pull/1359).

File: documentation/polymer-templates/tutorial-component-integration.asciidoc [hunk 6]
```diff
@@ -1,14 +1,14 @@
 ---
-title: PolymerTemplate, Detecting Component Mappings
-order: 14
+title: Detecting Component Mappings
+order: 18
```
Comment (pleku) [inline documentation/polymer-templates/tutorial-component-integration.asciidoc:3]: As this (also) applies for lit template, I think it should not be the last (or among the last) but before the polymer template specific pages, but as last of the (lit) template docs.

**Review by pleku — CHANGES_REQUESTED** [review]

File: documentation/polymer-templates/tutorial-component-integration.asciidoc (hunk 6 above)
Comment (szolo) [reply documentation/polymer-templates/tutorial-component-integration.asciidoc]: @pleku good point, fixed.

**Review by pleku — APPROVED** [review]

**Pull request merged**